vagrant_setup/go.sh

```
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+sudo curl -O https://storage.googleapis.com/golang/go1.18.1.linux-amd64.tar.gz
+sudo tar -xvf go1.18.1.linux-amd64.tar.gz
+sudo mv go /usr/local
+sudo echo 'export GOROOT=/usr/local/go' >> /etc/profile
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use .. | sudo tee -a file

(SC2024)